web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -384,11 +385,16 @@ function getExternalServices(path: AssetPath) {
           && _path.aggregate_size <= service.maxsize
   );
 
+  const baseApiUrl = process.env.VUE_APP_DANDI_API_ROOT;
+  const assetURL = () => (embargoed.value
+    ? `${baseApiUrl}assets/${path.asset?.asset_id}/download/`
+    : trimEnd((path.asset as AssetFile).url, '/'));
```

Comment:
Is there some particular reason to remove trailing slashes here? Shouldn't we just use whatever URL comes back from the API?

Reply:
I am not sure. I was just copying over the `trimEnd()` expression into this more expansive function. So if we want to change that, I'd say let's do it in a followup where we can properly analyze the effect of doing so. (For example, will it cause one of these dreaded and annoying redirects to the version with the trailing slash? etc.)

Reply:
My bad, I didn't notice that it existed before the change.